content/applications/general/in_app_purchase.rst

```diff
@@ -2,58 +2,142 @@
+   </applications/marketing/sms_marketing/pricing/pricing_and_faq>`.
 
-.. image:: in_app_purchase/image2.png
-   :align: center
+.. _iap/buying_credits:
 
-.. tip::
-   An IAP account can be disabled by appending `+disabled` to its token.
-   Reverting this change will re-enable the account.
+Buy credits
+-----------
+
+To manually buy credits:
 
-IAP Portal
-==========
+#. Open the :guilabel:`Settings app`.
+#. In the search bar, type :guilabel:`IAP`.
+#. Click :guilabel:`View My Services`.
+#. On the :guilabel:`IAP Account Settings` page, click on the service that needs more credits.
+#. On the :guilabel:`Account Information` page, click the :guilabel:`Buy Credit` button.
 
-The IAP Portal is a platform regrouping your IAP Services. It is accessible from
-:menuselection:`Settings app --> Odoo IAP --> View my Services`. From there, you can view your current
-balance, recharge your credits and set a reminder when your balance falls below a threshold.
+   - The balance of credits is displayed below the :guilabel:`Buy Credit` button.
+#. Click :guilabel:`Buy` on the desired pack of credits, and follow the prompts to enter payment
+   details.
+#. Once the transaction is complete, the credits will be available for use.
 
-.. image:: in_app_purchase/image3.png
+.. image:: in_app_purchase/account-info.png
    :align: center
+   :alt: The Account Information page for an IAP service showing the Buy Credit button.
+
+.. tip::
+   Users who have the Enterprise version of Odoo Online get free credits to test IAP features.
+
+.. _in_app_purchase/low-credits:
+
+Low-credit notification
+-----------------------
 
-Get notified when credits are low
-=================================
+To be notified when a service's credits are low:
 
-To be notified when it’s time to recharge your credits, you can go to your IAP Portal through
-:menuselection:`Settings app --> Odoo IAP --> View my Services`, unfold a service and check the
-Receive threshold warning option. Then, you can provide a minimum amount of credits and email
-addresses. Now, every time that the limit is reached, an automatic reminder will be sent by
-email!
+#. Open the :guilabel:`Settings app`.
+#. In the search bar, type :guilabel:`IAP`.
+#. Click :guilabel:`View My Services`.
+#. On the :guilabel:`IAP Account Settings` page, click on the service that needs an alert.
+#. On the :guilabel:`Account Information` page, click the :guilabel:`Warn Me` box. This causes two
+   more fields to appear.
+#. In the :guilabel:`Threshold` field, type an amount. Odoo will send a notification email when the
+   balance falls below this amount.
+#. In the :guilabel:`Email` field, type the email address that should receive the notification.
+#. Click the :guilabel:`Save Manually (cloud)` icon on the top-left of the page.
 
-.. image:: in_app_purchase/image4.png
+.. image:: in_app_purchase/low-credits.png
    :align: center
+   :alt: Odoo will send an email alert when credits for this service fall below the threshold.
```

Comment:
Optional-- add a section on how to manually trigger the IAP service if not already configured (use the service anywhere in the DB) and manually add the keys in if they are purchased outside the DB. 

Rare case but can occur during testing or migration.